homeworks/schreinerbalazs/3_conditionals_loops/flat_finder.py

```diff
@@ -0,0 +1,22 @@
+city = input("Melyik városba költöznél? ").strip()
+
+# Ha a város Washington, azonnal kiírjuk, hogy nem költözne oda
+if city.lower() == "washington":
+    print(f"Sarah inkább az utcán lakik de {city}-ba nem költözik!")
+else:
+    # Ha nem Washington, bekérjük a lakbér árát
+    rent = float(input("Mennyi a havi lakbér (USD)? "))
+
+    # Feltételek ellenőrzése
+    if city.lower() == "new york" or city.lower() == "san francisco":
```

Comment:
a lower()-t meghívhatnád akár már az első soron is:
`city = input("Melyik városba költöznél? ").strip().lower()`
így itt nem lenne szükség rá hogy kétszer is meghívd.

Egy másik egyszerűsítési lehetőség ha membership operatort használsz:
`if city in ["new york", "san francisco"]:`